### PR TITLE
update troubleshooting on mac

### LINKED ISSUE #3698: Request for Documentation: using mac's default archive utility prevents users from openning the app
Mattermost user `guillermo.vaya` from https://community-release.mattermost.com has requested the following be documented:

```
also, on some setups, using mac archive utility (mac's default for handling zips) causes it to not open, using a third party software (like keka or unarchiver) seems to fix that
```

See the original post [here](https://community-release.mattermost.com/_redirect/pl/9ze4ju3rzjfk9b86g3z341ji8y).

_This issue was generated from [Mattermost](https://mattermost.com) using the [Doc Up](https://github.com/jwilander/mattermost-plugin-docup) plugin._

### PR DESCRIPTION
**Pull request by Willyfrog**
#### Summary

adds troubleshooting for mac on catalina

#### Ticket Link

fixes #3698

### CODE REVIEW

File: source/install/desktop.rst [hunk 1]
```diff
@@ -126,6 +126,11 @@ Troubleshooting
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     
     The app data might be corrupted - remove all the files in ``%LOCALAPPDATA%\mattermost``, then try re-installing the app.
+    
+"The application "Mattermost" can't be opened" dialog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    On MacOs Catalina, this dialog can be triggered if using the default Mac Archive Utility. Using a third party arhchive tool like [Keka](https://www.keka.io/) or [Unarchiver](https://macpaw.com/the-unarchiver) fixes the problem.
```
Comment (justinegeffen) [inline source/install/desktop.rst:133]:
```suggestion
   On macOS Catalina, this dialog can be triggered if the Mac Archive Utility is the default method for decompressing files. In this case using a third-party tool such as [Keka](https://www.keka.io/) or [Unarchiver](https://macpaw.com/the-unarchiver) can resolve the problem.
```

**Comment by justinegeffen** [comment]
Thanks, @Willyfrog, this is really helpful. Some stylistic feedback but otherwise this is great. :) I appreciate you updating the docs!

**Comment by Willyfrog** [comment]
Definitely better, thanks!

File: source/install/desktop.rst [hunk 2]
```diff
@@ -126,6 +126,11 @@ Troubleshooting
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     
     The app data might be corrupted - remove all the files in ``%LOCALAPPDATA%\mattermost``, then try re-installing the app.
+    
+"The application "Mattermost" can't be opened" dialog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   On macOS Catalina, this dialog can be triggered if the Mac Archive Utility is the default method for decompressing files. In this case using a third-party tool such as [Keka](https://www.keka.io/) or [Unarchiver](https://macpaw.com/the-unarchiver) can resolve the problem.
```
Comment (justinegeffen) [inline source/install/desktop.rst:133]:
```suggestion
   On macOS Catalina, this dialog can be triggered if the Mac Archive Utility is the default method for decompressing files. In this case using a third-party tool such as [Keka](https://www.keka.io/) or [Unarchiver](https://macpaw.com/the-unarchiver) may resolve the problem.
```

**Comment by justinegeffen** [comment]
I changed, and committed, the order of "can" and "may". Otherwise it's good to go. :)

**Pull request merged**